Sources/Build/BuildDescription/SwiftModuleBuildDescription.swift

```diff
@@ -127,7 +127,11 @@ public final class SwiftModuleBuildDescription {
 
     var modulesPath: AbsolutePath {
         let suffix = self.buildParameters.suffix
-        return self.buildParameters.buildPath.appending(component: "Modules\(suffix)")
+        var path = self.buildParameters.buildPath.appending(component: "Modules\(suffix)")
+        if self.windowsTargetType == .exporting {
+            path = path.appending("exporting")
+        }
+        return path
```

Comment:
I really don't like how we're putting all the modules in a single directory and adding it to the include path. Since the exporting modules have the same name, I had to create another directory to put them in and add that include path for the products that consume them.

Reply:
yeah putting all the modules in the same directory is extra annoying because you can end up `import`ing things you haven't declared a dependency on.

Reply:
Question is, who do I break if I change this?